test.js

```diff
@@ -727,7 +727,8 @@ test('poetry py3.6 can package flask with default options', t => {
   sls(['package']);
   const zipfiles = listZipFiles('.serverless/sls-py-req-test.zip');
   t.true(zipfiles.includes(`flask${sep}__init__.py`), 'flask is packaged');
-  t.true(zipfiles.includes(`boto3${sep}__init__.py`), 'boto3 is packaged');
+  t.true(zipfiles.includes(`bottle.py`), 'bottle is packaged');
+  t.false(zipfiles.includes(`boto3${sep}__init__.py`), 'boto3 is NOT packaged');
```

Comment:
@jaysonsantos This assertion change you made seems incorrect given that the `pyproject.toml` has `boto3 = "^1.9"` specified. I would expect it to be packaged. Am I missing something about what this test is supposed to be checking?